sources/p987.6/sections/02-norm_refs.adoc

```diff
@@ -0,0 +1,35 @@
+
+[[cls_2]]
```

Comment:
Let's never assign anchors for bibliography sections.

Reply:
Ok, fixed.